src/main/resources/static/assets/js/toastr.js

```diff
@@ -52,9 +52,8 @@ $(function() {
             loaderBg:'#ff6849',
             icon: 'error',
             hideAfter: 3500
-            
+
           });
 
      });
 });
-          
```

Comment:
GitHub's UI is not helping review such changes as it seems the newline at the end of file was removed when in reality there were 2 of them and only 1 removed.